hack/istio/multicluster/env.sh

```diff
@@ -18,8 +18,6 @@ if [ "${HACK_ENV_DONE:-}" == "true" ]; then
   return 0
 fi
 
-set -u
```

Comment:
i don't think we want to get rid of this. This is a good validation feature because it helps catch instances where the code is using a variable that was never defined (or it catches a typo in a variable name). This has helped me catch many bugs that I otherwise would have missed.

If you expect a variable may not be defined in a particular place in the code, then simply use the syntax `${VAR:-}`.

Reply:
Thanks @jmazzitelli, I did not know the syntax  `${VAR:-}` for unbound variables. I have restored the `set -u` flag again.